modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/GeolocationField.testcase

```diff
@@ -30,6 +34,28 @@ definition {
 		else {
 			JSONGroup.deleteGroupByName(groupName = "Test Site Name");
 		}
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Web Content");
+
+		Navigator.gotoNavItem(navItem = "Structures");
+
+		Search.searchCP(searchTerm = "Structure");
+
+		while (IsElementNotPresent(locator1 = "Message#EMPTY_INFO")) {
+			PortletEntry.selectAll();
+
+			ClickNoError(locator1 = "Icon#DELETE");
+
+			AssertConfirm.assertConfirmationNoWait(value1 = "Are you sure you want to delete this? It will be deleted immediately.");
+
+			Alert.viewSuccessMessage();
+		}
+
+		Portlet.viewEmptyMessage(message = "There are no results.");
```

Comment:
Not needed. The structure is created in a new site that is deleted above. It is faster to create a site with JSON and delete it than iterate the UI deleting structures.